### ISSUE TITLE
Visualization of learned representation in Figure 2.

### ISSUE THREAD

**Issue by GoodMan0** [zwvews/LADG#1]
Thanks for your excellent work. I am confused about the details of the visualization results shown in Figure 2. 

1. Is each point represents the feature vector extracted from the backbone or the logit output from the classifier?
2. Are all data visualized, or only the training set(validation set) is shown in Figure 2?

The ERM visualization results seem to have satisfactory classification performance on the target domain(art-painting?), but I can not achieve this in my machine. Could you please release the visualization code and clear up my confusion?
